app.rb

```diff
@@ -3,9 +3,13 @@
 require 'json'
 require 'sinatra/base'
 require './lib/blockchain'
+require 'sinatra'
+require 'sinatra/flash'
 
 # Understands sending and receiving transaction hashes
 class BlockchainApp < Sinatra::Base
+  enable :sessions
```

Comment:
Did you end up needing to use this?

Reply:
this is needed for the flash messages to appear

Reply:
Awesome